inference-engine/src/transformations/src/transformations/common_optimizations/random_uniform_fusion.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/random_uniform_fusion.hpp"
+
+#include <memory>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include "itt.hpp"
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::RandomUniformFusion, "RandomUniformFusion", 0);
+NGRAPH_RTTI_DEFINITION(ngraph::pass::RandomUniformMulFusion, "RandomUniformMulFusion", 0);
+NGRAPH_RTTI_DEFINITION(ngraph::pass::RandomUniformAddFusion, "RandomUniformAddFusion", 0);
+
+ngraph::pass::RandomUniformMulFusion::RandomUniformMulFusion() {
+    MATCHER_SCOPE(RandomUniformMulFusion);
+    const auto data_pattern = ngraph::pattern::any_input();
```

Comment:
Please use `const auto &` instead of just `const auto` to avoid excess c-tors calls.